dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/EnrollmentImportValidationTest.java

```diff
@@ -91,6 +91,25 @@ void testEnrollmentValidationOkAll()
         assertEquals( TrackerStatus.OK, trackerImportReport.getStatus() );
     }
 
+    @Test
+    void testEnrollmentValidationFailMissingOrgUnitDeleteStrategy()
+        throws IOException
+    {
+
+        TrackerImportParams params = createBundleFromJson( "tracker/validations/enrollments_te_enrollments-data.json" );
+        params.setImportStrategy( TrackerImportStrategy.CREATE );
+        TrackerImportReport trackerImportReport = trackerImportService.importTracker( params );
+        assertEquals( 0, trackerImportReport.getValidationReport().getErrors().size() );
+        assertEquals( TrackerStatus.OK, trackerImportReport.getStatus() );
+
+        params = createBundleFromJson( "tracker/validations/enrollments_te_missing_ou.json" );
+        params.setImportStrategy( TrackerImportStrategy.DELETE );
+        trackerImportReport = trackerImportService.importTracker( params );
```

Comment:
can you please add newlines around the 

`trackerImportReport = trackerImportService.importTracker( params );`

so its clear on first sight what method we are testing.